spark/src/main/scala/io/delta/sharing/spark/DeltaSharingProfileProvider.scala

```diff
@@ -47,7 +47,10 @@ trait DeltaSharingProfileProvider {
 
   def getCustomTablePath(tablePath: String): String = tablePath
 
-  def getCustomRefresher(refresher: () => Map[String, String]): () => Map[String, String] = {
+  // Map[String, String] is the id to url map.
+  // Long is the minimum url expiration time for all the urls.
```

Comment:
Is this min across all urls, or just the expiry for the one?

Reply:
across all urls.

Reply:
it's in the comment.